code/src/utils/get-local-storage-cart/getLocalStorageCart.ts

```diff
@@ -0,0 +1,33 @@
+import { initialCart } from "@/constants/cart";
+import { LOCAL_STORAGE_KEYS } from "@/constants/common";
+import { CartType } from "@/types/cart.types";
+
+const getLocalStorageCart = (): CartType => {
```

Comment:
is that return type required here? Probably it's generating automatically

Reply:
I think type is needed because value can come from `localStorage` in it is any by default